Src/Support/Google.Apis.Auth/OAuth2/CredentialFactory.cs

```diff
@@ -336,6 +336,7 @@ private static ImpersonatedCredential CreateImpersonatedServiceAccountCredential
         {
             DelegateAccounts = parameters.Delegates?.Length > 0 ? parameters.Delegates.ToList() : null,
             QuotaProject = parameters.QuotaProject,
+            Scopes = parameters.Scopes?.Length > 0 ? parameters.Scopes.ToList() : null,
```

Comment:
We shouldn't need ToList, Scopes is an `IEnumerable<string>` so the array should be fine.

```suggestion
            Scopes = parameters.Scopes?.Length > 0 ? parameters.Scopes : null,
```

Reply:
I've updated here and I also updated for `DelegateAccounts` to keep things consistent.